conf/messages

```diff
@@ -2,8 +2,21 @@
 servicename.titleSuffix=- Agent services account - GOV.UK
 
 #Connect agent services
-connectAgentServices.title=Connect to your agent services account
-connectAgentServices.label.accountNumber=Agent services account number
+enter-account-number.title=What is your agent services account number?
+enter-account-number.details.heading=For example, XARN-123-4567
```

Comment:
We may not accept the format mentioned in the example ARN. I think we need to remove `-`.